spec/models/post_mover_spec.rb

```diff
@@ -47,16 +55,21 @@
     context 'success' do
 
       it "enqueues a job to notify users" do
-        topic.stubs(:add_moderator_post)
+        Jobs.stubs(:enqueue)
```

Comment:
We generally try to avoid stubbing methods without the parameters since it makes the test brittle if we need some other job to be en-queued in the future.